kwave/utils/kwave_array.py

```diff
@@ -263,10 +264,10 @@ def add_rect_element(self, position, Lx, Ly, theta):
 
     def add_arc_element(self, position, radius, diameter, focus_pos):
 
-        assert isinstance(position, (list, tuple)), "'position' must be list or tuple"
+        assert isinstance(position, (list, tuple, Vector)), "'position' must be list, tuple or Vector"
         assert isinstance(radius, (int, float)), "'radius' must be an integer or float"
         assert isinstance(diameter, (int, float)), "'diameter' must be an integer or float"
-        assert isinstance(focus_pos, (list, tuple)), "'focus_pos' must be list or tuple"
+        assert isinstance(focus_pos, (list, tuple, Vector)), "'focus_pos' must be list, tuple or Vector"
```

Comment:
Why not to check against `Iterable`?

Reply:
I have changed this, but I am not used to using Iterable